apps/health_alerts/constants.py

```diff
@@ -0,0 +1,24 @@
+ANIMAL_VETERINARY_CLINICS = "AV"
```

Comment:
Another way to do this is to use Django's enumeration types and include this a class inside the model class, which encapsulates things a little better IMO: https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types

But it is fine to do it like this too.

Reply:
Thanks, will do this going forward!

Reply:
Implemented. done.